### PR TITLE
Add steps to createAnswer and explicitly specify what is queued

### PR DESCRIPTION
**Pull request by adam-be**
These changes are dependent on the isClosed checks added in PR #745

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -1335,23 +1335,102 @@
...
+                                <p>If <var>connection</var>'s [[<a>isClosed</a>]]
+                                slot is <code>true</code>, then abort these
+                                steps.</p>
+                              </li>
+                              <li>
+                                <p>If the identity provider was unable to
+                                produce an identity assertion, reject
+                                <var>p</var> with a <code>DOMException</code>
+                                object whose <code>name</code> attribute has
+                                the value <code>NotReadableError</code>, and
```
Comment (fluffy) [inline webrtc.html]: Be nice to get more information about what failed

Comment (adam-be) [inline webrtc.html]: Yes. New issue?

**Comment by adam-be** [comment]
@jan-ivar, this PR is quite similar to the one about createOffer. Would you like to have a look?

File: webrtc.html [hunk 2]
```diff
@@ -1335,23 +1335,101 @@
...
+                  <li>
+                    <p>Return the result of <a href=
+                    "#enqueue-an-operation">enqueuing</a> the following
+                    steps:</p>
+                    <ol>
+                      <li>
+                        <p>If <code><a data-for=
+                        "RTCPeerConnection">remoteDescription</a></code> is
+                        <code>null</code> return a promise rejected with an
+                        <code>InvalidStateError</code>.</p>
```
Comment (jan-ivar) [inline webrtc.html]: Again I would suggest doing input validation before enqueuing.

File: webrtc.html [hunk 3]
```diff
@@ -1335,23 +1335,101 @@
...
+                              <li>
+                                <p>Let <var>answer</var> be a newly created
+                                <code><a>RTCSessionDescriptionInit</a></code>
+                                dictionary with its <code>type</code> member
+                                initialized to the string <code>"answer"</code>
+                                and its <code>sdp</code> member initialized to
+                                <var>sdpString</var>.</p>
+                              </li>
+                              <li>
+                                <p>If not all (non-stopped) <code>
```
Comment (jan-ivar) [inline webrtc.html]: Unless all? And maybe lose parens?

**Comment by jan-ivar** [comment]
Lgtm.

**Pull request merged**